docs/questions/native-editor/writing-sql.md

```diff
@@ -77,6 +77,20 @@ Make sure your SQL dialect matches the database you've selected. Common errors:
 
 For more help, see [Troubleshooting SQL error messages](../../troubleshooting-guide/error-message.md#sql-editor).
 
+## Explore SQL question results using the Query Builder
+
+On saved SQL questions without [parameters](./sql-parameters.md), you'll get the **Explore results** button. It will create a new Query Builder question that uses the SQL question results as a data source.
+
+![explore restuls button](../images/explore-results.png)
+
+## Turn a SQL question into a model to enable drill-through
```

Comment:
```suggestion
## To enable drill-through, turn a SQL question into a model and set the data types
```

It's a bit long, though we should first start with what, then the how. And the types are important in this case, as turning it into a model alone is insufficient.